docs/pages/ktable.vue

```diff
@@ -210,8 +232,141 @@
+      </p>
+
+      <p>
+        The <code>defaultSort</code> attribute takes an object with two properties - <code>columnId</code> and <code>direction</code>. The <code>columnId</code> is the unique identifier of the column based on which the table should be sorted. The <code>direction</code> can be either <code>asc</code> or <code>desc</code>.
+      </p>
+
+      <p>
+        To make use of <code>defaultSort</code>, please ensure that the <code>disableBuiltinSorting</code> attribute is not set to <code>true</code>.
+      </p>
+
+      <DocsShowCode language="html">
+        <h4>Sortable Table with Rows Sorted by 'Age' Column</h4>
+        <KTable
+          :headers="headers"
+          :rows="rows"
+          caption="Sortable Table with Rows Sorted by 'Age' Column"
+          sortable
+          :defaultSort="{ columnId: 'age', direction: 'asc' }"
+        />
+
+        <h4>Unsortable Table with Rows Sorted by 'Age' Column</h4>
+        <KTable 
+          :headers="headers"
+          :rows="rows"
+          caption="Unsortable Table with Rows Sorted by 'Age' Column"
+          :defaultSort="{ columnId: 'age', direction: 'asc' }"
+        />
+
+      </DocsShowCode>
+
+      <!-- eslint-disable -->
+      <DocsShowCode language="javascript">
+        data() {
+          return {
+            headers: [
+              { label: 'Name', dataType: 'string', columnId: 'name' },
+              { label: 'Age', dataType: 'number', columnId: 'age' },
+              { label: 'City', dataType: 'string', columnId: 'city' },
+            ],
+            rows: [
+              ['John Doe', 28, 'New York'],
+              ['Jane Smith', 34, 'Los Angeles'],
+              ['Samuel Green', 22, 'Chicago'],
+              ['Alice Johnson', 30, 'Houston'],
+              ['Michael Brown', 45, 'Phoenix'],
+              ['Emily Davis', 27, 'Philadelphia'],
+            ]
+          };
+        },
+      </DocsShowCode>
+      <!-- eslint-enable -->
+
+      <DocsShow block>
+        <h4>Sortable Table with Rows Sorted by 'Age' Column</h4>
+        <KTable
+          :headers="headers"
+          :rows="rows"
+          caption="Sortable Table with Rows Sorted by 'Age' Column"
+          sortable
+          :defaultSort="{ columnId: 'age', direction: 'asc' }"
+        />
+
+        <h4>Unsortable Table with Rows Sorted by 'Age' Column</h4>
+        <KTable
+          :headers="headers"
+          :rows="rows"
+          caption="Unsortable Table with Rows Sorted by 'Age' Column"
+          :defaultSort="{ columnId: 'age', direction: 'asc' }"
+        />
+      </DocsShow>
+
+      <!-- Disable Builtin Sorting -->
+      <h3>Disable Builtin Sorting</h3>
+      <p>
+        You can make use of the <code>disableBuiltinSorting</code> attribute to disable all sorting functionality by the table component. This is useful when you want to display the data in a particular order or you want to define a custom sorting function. 
```

Comment:
Very nice, thanks for taking care of documenting this. Few tweaks to improve with clarity:

```suggestion
        For <code>sortable</code> tables, you can use the <code>disableBuiltinSorting</code> prop to disable built-in sort function. This is useful when the table receives already sorted data, for example when sorting is done by backend or a custom sorting function outside the table. In this case, when one of the header sort buttons is clicked, the table won't sort the column itself, but only emit the <code>changeSort</code> event to notify the parent component to handle the sorting logic. The event contains column index of the header and the sort order in its payload.
```